docs/guides/verify-smart-contract.md

```diff
@@ -0,0 +1,145 @@
+---
+id: verify-smart-contract
+title: Verifying Smart Contracts
+sidebar_label: Verify Contracts
+description: A guide to verifying smart contracts in various frameworks on Polygon zkEVM network.
+keywords:
+  - docs
+  - polygon
+  - developer
+  - guides
+  - verify smart contract
+  - zkEVM
+---
+
+Once a smart contract is deployed to zkEVM, it can be verified in various ways depending on the framework of deployment as well the complexity of the contract. The aim here to use examples to illustrate how you can manually verify a deployed smart contract. 
```

Comment:
```suggestion
Once a smart contract is deployed to zkEVM, it can be verified in various ways depending on the framework of deployment as well as the complexity of the contract. The aim here is to use examples to illustrate how you can manually verify a deployed smart contract. 
```